### PR TITLE
docs: read replicas

### LINKED ISSUE #5441: Docs: Read Replica support

### PR DESCRIPTION
**Pull request by ruheni**
Closes #5441

### CODE REVIEW

**Review by nikolasburk — APPROVED** [review]
looks good to me, let's merge? (if we still find issues in this, we can also correct them )

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (jharrell) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]: I think we should change this to be action focused. "Learn how to set up" rather than "this page explains how to set up"
```suggestion
metaDescription: 'Learn how to set up and use read replicas with Prisma Client'
```

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (jharrell) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]:
```suggestion
Read replicas enable you to distribute workloads across database replicas for high-traffic workloads. The [read replicas extension](https://github.com/prisma/extension-read-replicas), `@prisma/extension-read-replicas`, adds support for read-only database replicas to Prisma Client.
```

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (jharrell) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]: meta comment. "read replicas extension" or "read replica extension"?

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (jharrell) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]:
```suggestion
Initialize the extension by extending your Prisma Client instance and providing the extension a connection string that points to your read replica in the `url` option of the extension.
```
not 100% on the terminology but something like this

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (jharrell) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]:
```suggestion
With the above setup, all read operations, e.g. `findMany`, will be executed against the database replica. All write operations, e.g. `create`, `update`, and `$transaction`, will be executed against your primary database.
```
editing for consistency and added an oxford comma. If that's against our style guide feel free to remove.

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (jharrell) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]: I think we should change this. We state above that the extension only supports Prisma v5.2.0+ so I don't think this admonition is necessary. Maybe we remove the above mention and move this admonition to the top of the page?

**Review by jharrell — APPROVED** [review]
Approving, but left some comments that I personally think should be changed. Happy to start a discussion. 😄

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (ruheni) [reply content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]: "read replicas extension" — based on the name of the repo 🙂

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]:
```suggestion
```

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]: link to release notes for version so one can get to those?

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx:26]: Why is this needed?

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]:
```suggestion
Initialize the extension by extending your Prisma Client instance and provide the extension a connection string that points to your read replica in the `url` option of the extension.
```

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]: `$transaction` itself is not necessarily a write operation, but more one that _can_ also include writes. Worth differentiating?

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]: This reads more like from an announcement blog post, not a documentation page.

**Review by janpio — APPROVED** [review]
Looks good, but let's wait for review from the expert, @SevInf, before merging.

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]:
```suggestion
The read replicas extension supports Prisma versions [5.2.0](https://github.com/prisma/prisma/releases/tag/5.2.0) and higher.
```

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (ruheni) [reply content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx:26]: Maintain the formatting of the code snippet below :)

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (ruheni) [reply content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]: Any suggestions on information that might be missing or improve the page to make it read like a docs page? Or is it line 84 that gives you the feeling it's an announcement blog post? Feedback and suggestions are welcome.

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]:
```suggestion
With the above setup, all read operations, e.g. `findMany`, will be executed against the database replica. All write operations — e.g. `create`, `update` — and `$transaction` queries, will be executed against your primary database.
```

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]: The comment is on line 84, which is under a headline but is general content. "Try it out" is also not docs-like.

Comment (ruheni) [reply content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]: How about:
"If you run into a bug or have feedback, create a GitHub issue [here](https://github.com/prisma/extension-read-replicas/issues/new)."

Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]: Yes, but then it still needs to be put into a location that is not under the last headline about `$replica`.

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]:
```suggestion
All read operations, e.g. `findMany`, will be executed against the database replica with the above setup. All write operations — e.g. `create`, `update` — and `$transaction` queries, will be executed against your primary database.

If you run into a bug or have feedback, create a GitHub issue [here](https://github.com/prisma/extension-read-replicas/issues/new).
```

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]:
```suggestion
The read replicas extension supports Prisma versions [5.2.0](https://github.com/prisma/prisma/releases/tag/5.2.0) and higher. If you run into a bug or have feedback, create a GitHub issue [here](https://github.com/prisma/extension-read-replicas/issues/new).
```

File: content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx
Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx]:
```suggestion
```

**Review by janpio — APPROVED** [review]

**Pull request merged**
